search-app/src/cmr/search/api/core.clj

```diff
@@ -0,0 +1,109 @@
+(ns cmr.search.api.core
+  "Core functions used by multiple API/routes namespaces."
+  (:require
+   [cmr.common-app.api.routes :as common-routes]
+   [cmr.common-app.services.search :as search]
+   [cmr.common.cache :as cache]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.services.errors :as svc-errors]
+   [cmr.umm-spec.versioning :as umm-version]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Constants, Utilities, and Support Functions
+
+(def search-result-supported-mime-types
+  "The mime types supported by search."
+  #{mt/any
+    mt/xml
+    mt/json
+    mt/umm-json
+    mt/umm-json-results
+    mt/legacy-umm-json
+    mt/echo10
+    mt/dif
+    mt/dif10
+    mt/atom
+    mt/iso19115
+    mt/opendata
+    mt/csv
+    mt/kml
+    mt/native})
+
+(defn- add-scroll-id-to-cache
```

Comment:
There's a `get-scroll-id-from-cache` in another namespace now. I think the pair of these functions should be in the same namespace.

Reply:
+1

Reply:
Moved.